Documentation/templates/strings/flat-swift4.md

```diff
@@ -21,7 +21,7 @@ You can customize some elements of this template by overriding the following par
 | `enumName` | `L10n` | Allows you to change the name of the generated `enum` containing all string tables. |
 | `noComments` | N/A | Setting this parameter will disable the comments describing the translation of a key. |
 | `publicAccess` | N/A | If set, the generated constants will be marked as `public`. Otherwise, they'll be declared `internal`. |
-| `localizeFunction` | `NSLocalizedString` | Allows you to set your own custom localization function. Your custom function must have the same signature as the one provided by `Foundation`, i.e. `yourFunctionName(_:tableName:bundle:comment:)` |
+| `lookupFunction` | N/A | Allows you to set your own custom localization function. You must provide the complete function signature (including parameters), and your function accepts 2 parameters: the localization key (`String`), and the localization table (`String`). E.g. `yourFunctionName(forKey:table:)` |
```

Comment:
```suggestion
| `lookupFunction` | N/A | Allows you to set your own custom localization function. The function needs to accepts 2 parameters: the localization key (`String`), and the localization table (`String`). If the function has named parameters, you must provide the complete function signature, including those named parameters – e.g. `yourFunctionName(forKey:table:)` |
```

Reworded because if the function takes unnamed parameters, it's ok to use `myFunction` (it's also ok to pass `myFunction(_:_:)` but not needed). wdyt? (if you're ok with that new wording don't forget to apply the same change suggestion to the 3 other documentation files)